pkg/app/execontext.go

```diff
@@ -106,7 +109,17 @@ func (ref *Output) Prompt(data string) {
 	color.Set(color.FgHiRed)
 	defer color.Unset()
 
-	fmt.Printf("cmd=%s prompt='%s'\n", ref.CmdName, data)
+	if ref.JSONFlag == "json" {
+		//marshal data to json
```

Comment:
We need to have a json equivalent of what's printed when the output is text where we print two key/value pairs: `cmd` and `prompt`.

Here we can use a generic map of strings and then set the `cmd` and `prompt` fields there. Something like this:

```
msg := map[string]string{
"cmd": ref.CmdName,
"prompt": data,
}
```

Then we'd json marshal `msg` and then print it to console.

Reply:
Done, thanks so much!